src/lib/InputProcessor.ts

```diff
@@ -80,3 +112,38 @@ function outputFormatFromInputs(inputs: Inputs): OutputFormat {
 		return OutputFormat.TABLE;
 	}
 }
+
+function trimMethodSpecifier(targetPath: string): string {
+	return targetPath.split('#')[0];
+}
+
+function getFirstCommonParentFolder(targetFiles: string[]) {
+	const longestCommonStr: string = getLongestCommonPrefix(targetFiles);
+	const commonParentFolder = getParentFolderOf(longestCommonStr);
+	return commonParentFolder.length == 0 ? '/' : commonParentFolder;
```

Comment:
Question: Should we maybe throw an error if no common folder can be found? Returning `/` is technically correct based on the input, but I feel like we should assume that any such case is a mistake.

Reply:
I went back and forth on this. I think that the fact we spit our a message to the log to indicate what we are using is fine here. It is also very rare to hit this... you would have to have things inside of your root. This will like never happen. So, since I thoroughly tested this I'm of the mindset that an error message would be overkill here.

Reply:
Reasonable. Might be worth adding a comment in the code detailing that rationale.